### PR TITLE
fix(DataGrid): virtual scrollbar not visible

### PR DESCRIPTION
**Pull request by Ratheeshrajan**
Contributes to #2390 

The virtual scroll bar (ie for Datagrids that use the `useInfiniteScroll` hook), is not visible.

#### What did you change?
Datagrid/styles/_datagrid.scss
Datagrid/Datagrid/DatagridVirtualBody.js
Datagrid/Datagrid/DatagridContent.js
#### How did you test and verify your work?
Storybook

### CODE REVIEW

**Comment by Ratheeshrajan** [comment]
@matthewgallo fixed the issue header not syncing when using horizontal virtual scrollbar.

**Comment by Ratheeshrajan** [comment]
V-11 PR added https://github.com/carbon-design-system/ibm-cloud-cognitive/pull/2467

File: packages/cloud-cognitive/src/components/Datagrid/styles/_datagrid.scss [hunk 1]
```diff
@@ -536,6 +536,22 @@
...
+.#{$block-class}__virtual-scrollbar::-webkit-scrollbar-track {
+  background: $layer;
+}
+
+.#{$block-class}__virtual-scrollbar::-webkit-scrollbar-thumb {
+  border-radius: 5px;
+}
+
+.#{$block-class}__virtualScrollContainer {
+}
```
[the remark this answers is not in the document]
Comment (elycheea) [reply packages/cloud-cognitive/src/components/Datagrid/styles/_datagrid.scss]: Actually, after checking the v11 branch, looks like we’re missing `width: 100%` here.

**Review by elycheea — APPROVED** [review]
LGTM.
@AlexanderMelox I know we wanted to have the px values changed to rems, but this might be a case where we could keep them as is since they’re meant for scrollbars — I don’t think we’d expect scrollbars to scale with text so I think pixels could be fine here.

**Pull request merged**